pom.xml

```diff
@@ -585,7 +619,36 @@
           <execution>
             <id>attach-javadocs</id>
             <configuration>
+              <sourcepath>${project.build.directory}/javadoc-generated-sources</sourcepath>
+              <doclet>org.commonjava.maven.ext.annotation.JavadocFilter</doclet>
+              <docletArtifact>
+                <groupId>org.commonjava.maven.ext</groupId>
+                <artifactId>pom-manipulation-annotation</artifactId>
+                <version>${project.version}</version>
```

Comment:
I see, because it's a dependency inside a plugin (execution/)configuration?

Reply:
`docletArtifact` needs a full GAV

Reply:
I mean is it the reason it uses `${project.version}` vs. `3.9-SNAPSHOT`.

Reply:
And for that matter, it should use a real version like 3.9-SNAPSHOT

Reply:
On second thoughts after testing it - it doesn't subsitute the docletArtifact version so I won't change that.